### PR TITLE
updates based on code review from Mike

### PR DESCRIPTION
**Pull request by baripembo**

### CODE REVIEW

File: src/hdx/scraper/iom/iom.py [hunk 1]
```diff
@@ -52,8 +60,13 @@ def scrape_data(self) -> list:
         """
         logger.info("Scraping data")
         data_url = self._configuration["base_url"]
+
+        start_year = 2014
```
Comment (mcarans) [inline src/hdx/scraper/iom/iom.py:64]: Should this be fixed or should it roll forward each year?

Comment (mcarans) [reply src/hdx/scraper/iom/iom.py:64]: GitHub doesn't allow me to comment on unchanged lines but "list" should be "List" further down:
def generate_dataset(self, data_by_year_list: List) -> Optional[Dataset]:

Comment (mcarans) [reply src/hdx/scraper/iom/iom.py:64]: Also in the __main__.py file, this should be removed (it can be set by environment variable):
hdx_site="dev",

**Review by mcarans — APPROVED** [review]
This looks great Erika!

File: src/hdx/scraper/iom/iom.py (hunk 1 above)
Comment (baripembo) [reply src/hdx/scraper/iom/iom.py:64]: > Should this be fixed or should it roll forward each year?

2014 appears to be the first year with data so I thought the dataset was meant to include all data but I will confirm, thanks!

**Pull request merged**